.github/workflows/ci.yml

```diff
@@ -154,7 +133,12 @@ jobs:
         if: github.ref == 'refs/heads/main'
         uses: changesets/action@v1
         with:
-          publish: npm run release
+          publish: |
+            npm run changeset-version
+            cd packages/python && uv publish --trusted-publishing always
         env:
           GITHUB_TOKEN: ${{ steps.generate-token.outputs.token }}
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+          COLUMNS: 150
+          UV_PYTHON: 3.13
+          UV_FROZEN: "1"
```

Comment:
I don't think you need this, it's set globally.

Reply:
removed